### PR TITLE
6.4.4 Hotfix documentation

### PR DESCRIPTION
**Pull request by mvalin**
### Description

- Duplicate directory 6.4.3 as 6.4.4
- Change the version and date
- Change sha256sum for the new installer version
- Add public documentation link

### CODE REVIEW

File: 6.X/6.4.4/03-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+As with installs, it can be used in airgapped or non-airgapped environments.
+
+Review the [Prerequisites](../README.md#prerequisites) and [Installation
+Options](../README.md#quickstart-install) for more context.
+
+## Upgrade Steps
+
+<br />
+
+### Step 1
```
Comment (radhikapc) [inline 6.X/6.4.4/03-upgrade.md:31]: Please give some meaningful heading.

File: 6.X/6.4.4/04-advanced_configuration.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+hostPathCustomPaths:
+  cassandra: /sysdig/cassandra
+  elasticsearch: /sysdig/elasticsearch
+  mysql: /sysdig/mysql
+  postgresql: /sysdig/postgresql    
+```
+
+## Installer on EKS
+
+### Creating a cluster
```
Comment (radhikapc) [inline 6.X/6.4.4/04-advanced_configuration.md:78]:
```suggestion
### Creating a Cluster
```

File: 6.X/6.4.4/01-command_line_arguments.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
+<!-- Space: TOOLS -->
+<!-- Parent: Installer -->
+<!-- Title: Command Line Arguments -->
+<!-- Layout: plain -->
+
+# Command line arguments
```
Comment (radhikapc) [inline 6.X/6.4.4/01-command_line_arguments.md:6]:
```suggestion
# Command Line Arguments
```

File: 6.X/6.4.4/04-advanced_configuration.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
+<!-- Space: TOOLS -->
+<!-- Parent: Installer -->
+<!-- Title: Advanced Configuration -->
+<!-- Layout: plain -->
+
+# Advanced configuration
```
Comment (radhikapc) [inline 6.X/6.4.4/04-advanced_configuration.md:6]:
```suggestion
# Advanced Configuration
```

File: 6.X/6.4.4/04-advanced_configuration.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+eksctl create cluster \
+   --name=eks-installer1 \
+   --node-type=m5.4xlarge \
+   --nodes=3 \
+   --version 1.14 \
+   --region=us-east-1 \
+   --vpc-public-subnets=<subnet1,subnet2>
+```
+
+### Additional installer configurations
```
Comment (radhikapc) [inline 6.X/6.4.4/04-advanced_configuration.md:92]:
```suggestion
### Additional Installer Configurations
```

File: 6.X/6.4.4/04-advanced_configuration.md [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+docker run  \
+  -v ~/.aws:/.aws \
+  -e HOST_USER=$(id -u) \
+  -e KUBECONFIG=/.kube/config \
+  -v ~/.kube:/.kube:Z \
+  -v $(pwd):/manifests:Z \
+  quay.io/sysdig/installer:<InstallerVersion>
+```
+
+### Running airgapped EKS
```
Comment (radhikapc) [inline 6.X/6.4.4/04-advanced_configuration.md:107]:
```suggestion
### Running Airgapped EKS
```

File: 6.X/6.4.4/04-advanced_configuration.md [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+
+```bash
+EKS=true bash sysdig_installer.tar.gz
+```
+
+The above ensures the `~/.aws` directory is correctly mounted for the airgap
+installer container.
+
+### Exposing the Sysdig endpoint
```
Comment (radhikapc) [inline 6.X/6.4.4/04-advanced_configuration.md:116]:
```suggestion
### Expose the Sysdig Endpoint
```

File: 6.X/6.4.4/04-advanced_configuration.md [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+
+Get the external IP/endpoint for the ingress service.
+
+```bash
+kubectl -n <namespace>  get service haproxy-ingress-service
+```
+
+In route53, create an A record with the DNS name pointing to the external IP/endpoint.
+
+### Gotchas
```
Comment (radhikapc) [inline 6.X/6.4.4/04-advanced_configuration.md:126]:
```suggestion
### Guidelines
```

File: 6.X/6.4.4/04-advanced_configuration.md [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+
+Make sure that subnets have an internet gateway configured and have a sufficient number of IP addresses.
+
+## Airgapped installations
```
Comment (radhikapc) [inline 6.X/6.4.4/04-advanced_configuration.md:130]:
```suggestion
## Airgapped Installations
```

File: 6.X/6.4.4/04-advanced_configuration.md [hunk 10]
```diff
@@ -0,0 +1,175 @@
...
+
+### Updating the feeds database in airgapped environments
```
Comment (radhikapc) [inline 6.X/6.4.4/04-advanced_configuration.md:132]:
```suggestion
### Update Database Feeds
```

File: 6.X/6.4.4/05-networkPolicies.md [hunk 11]
```diff
@@ -0,0 +1,107 @@
+<!-- Space: TOOLS -->
+<!-- Parent: Installer -->
+<!-- Title: Network policies v2 in the installer -->
+<!-- Layout: plain -->
+
+# Network policies
```
Comment (radhikapc) [inline 6.X/6.4.4/05-networkPolicies.md:6]:
```suggestion
# Network Policies
```

**Review by radhikapc — CHANGES_REQUESTED** [review]
nits

File: 6.X/6.4.4/03-upgrade.md (hunk 1 above)
Comment (darroyo-sysdig) [reply 6.X/6.4.4/03-upgrade.md:31]: What should the rest of the steps be named?

Comment (mvalin) [reply 6.X/6.4.4/03-upgrade.md]: Already set titles plaase check.

File: 6.X/6.4.4/03-upgrade.md [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+
+Copy the current version of `sysdig-chart/values.yaml` to your working directory.
+
+```bash
+wget https://raw.githubusercontent.com/draios/sysdig-cloud-scripts/installer/installer/values.yaml
+```
+
+<br />
+
+### Customize the `values.yaml`
```
Comment (radhikapc) [inline 6.X/6.4.4/03-upgrade.md:41]:
```suggestion
### Customize `values.yaml`
```

File: 6.X/6.4.4/03-upgrade.md [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+As with installs, it can be used in airgapped or non-airgapped environments.
+
+Review the [Prerequisites](../README.md#prerequisites) and [Installation
+Options](../README.md#quickstart-install) for more context.
+
+## Upgrade Steps
+
+<br />
+
+### Download base template `values.yaml`
```
Comment (radhikapc) [inline 6.X/6.4.4/03-upgrade.md:31]:
```suggestion
### Download base template 
```

**Review by radhikapc — CHANGES_REQUESTED** [review]
nits :-)

**Review by radhikapc — APPROVED** [review]

**Pull request merged**